Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -3585,6 +3588,37 @@ extension BrowserViewController: UNUserNotificationCenterDelegate {
   }
 }
 
+// MARK: UIScreenshotServiceDelegate
+
+extension BrowserViewController: UIScreenshotServiceDelegate {
+  public func screenshotService(_ screenshotService: UIScreenshotService, generatePDFRepresentationWithCompletion completionHandler: @escaping (Data?, Int, CGRect) -> Void) {
```

Comment:
You can use the `async-await` version I think. But just a `nit:` so no worries.

Reply:
let's go Soner :D